src/components/onboarding/Intro/index.tsx

```diff
@@ -1,27 +1,155 @@
+
+  useEffect(() => {
+    const handleMouseMove = (e: PointerEvent) => {
+      if (!ref.current) {
+        return;
+      }
+      const { left, top } = ref.current.getBoundingClientRect();
+      setMouseX(e.pointerType === 'mouse' ? e.clientX - left : 0);
+      setMouseY(e.pointerType === 'mouse' ? e.clientY - top : 0);
+    };
+    document.addEventListener('pointermove', handleMouseMove);
+    return () => {
+      document.removeEventListener('pointermove', handleMouseMove);
+    };
+  });
+
   return (
     <div className={styles.wrapper}>
-      <Image
-        src={TempImage}
-        alt="temp image"
-        width={256}
-        height={256}
-        className={`${styles.image} ${styles.desktopOnly}`}
-        style={{ animationDelay: '0.2s' }}
-      />
-      <Image src={TempImage} alt="temp image" width={256} height={256} className={styles.image} />
-      <Image
-        src={TempImage}
-        alt="temp image"
-        width={256}
-        height={256}
-        className={`${styles.image} ${styles.desktopOnly}`}
-        style={{ animationDelay: '0.4s' }}
-      />
+      <div className={styles.anchor} ref={ref}>
+        {images.map(({ src, alt, desktopSize, mobileSize, round }, i) => (
+          <div
+            key={src.src}
+            className={styles.imageWrapper}
+            style={{
+              transform: `translate(${mouseX / (10 + images.length - i)}px, ${
+                mouseY / (10 + images.length - i)
+              }px)`,
+            }}
+          >
+            <Image
+              className={`${styles.image} ${round ? styles.pill : ''} ${styles.desktopOnly}`}
+              src={src}
+              alt={alt}
+              width={desktopSize[2]}
+              height={desktopSize[3]}
+              style={{
+                left: `${desktopSize[0] - DESKTOP_OFFSET_X}px`,
+                top: `${desktopSize[1] - DESKTOP_OFFSET_Y}px`,
+                transformOrigin: `${DESKTOP_OFFSET_X - desktopSize[0]}px ${
+                  DESKTOP_OFFSET_Y - desktopSize[1]
+                }px`,
+                animationDelay: `${i * 0.1 + 0.5}s`,
+                animationDuration: `${i * 0.05 + 1}s`,
+              }}
+            />
+            <Image
+              className={`${styles.image} ${round ? styles.pill : ''} ${styles.mobileOnly}`}
+              src={src}
+              alt={alt}
+              width={mobileSize[2]}
+              height={mobileSize[3]}
+              style={{
+                left: `calc(${mobileSize[0] - MOBILE_OFFSET_X}px + ${
+                  (mobileSize[0] - MOBILE_OFFSET_X + mobileSize[2] / 2) * 0.2
+                }vw)`,
+                top: `${mobileSize[1] - MOBILE_OFFSET_Y}px`,
+                transformOrigin: `${MOBILE_OFFSET_X - mobileSize[0]}px ${
+                  MOBILE_OFFSET_Y - mobileSize[1]
+                }px`,
+                animationDelay: `${i * 0.1 + 0.5}s`,
+                animationDuration: `${i * 0.05 + 1}s`,
+              }}
```

Comment:
This is fine, but let's try to refactor out what we can to make this more readable if possible. I see animation stuff is getting reused, and a lot of logic is basically the same except switching between mobile and desktop. 

Maybe a helper function that returns the args, then we can just spread them here?